codalab/apps/web/templates/web/competitions/view.html

```diff
@@ -229,54 +229,29 @@ <h3 class="panel-title"><strong>Competition Ends</strong></h3>
         </div>
     </section>
         <div class="row">
-        <div class="col-md-4">
-            {% if top_three_leaders and not active_phase.is_blind %}
-                <div class="panel">
-                    <div class="panel-body">
-                        <center><h4>Top Three</h4></center>
-                        <table class="table table-responsive">
-                            <thead>
-                                <tr>
-                                    <th>Rank</th>
-                                    <th>Username</th>
-                                    <th>Score</th>
-                                </tr>
-                            </thead>
-                            <tbody>
-                                {% for leader in top_three_leaders %}
-                                    <tr>
-                                        <td class="leader">{{ forloop.counter }}</td>
-                                        <td class="leader">{{ leader.username }}</td>
-                                        <td class="leader">{{ leader.score }}</td>
-                                    </tr>
-                                {% endfor %}
-                            </tbody>
-                        </table>
-                    </div>
-                </div>
-            {% else %}
-                <div class="panel">
-                    <div class="panel-body"">
-                        <center><h4>Top Three</h4></center>
-                        <table class="table table-responsive">
-                            <thead>
-                                <tr>
-                                    <th>Rank</th>
-                                    <th>Username</th>
-                                    <th>Score</th>
-                                </tr>
-                            </thead>
-                            <tbody>
-                                <tr>
-                                    <td>No data</td>
-                                </tr>
-                            </tbody>
-                        </table>
-                    </div>
-                </div>
-            {% endif %}
+        <div class="col-sm-4">
+            <table class="table">
+                <thead>
+                <tr>
+                    <th>#</th>
+                    <th>Username</th>
+                    <th>Score</th>
+                </tr>
+                </thead>
+                <tbody>
+                {% if not active_phase.is_blind %}
+                    {% for leader in top_three %}
+                        <tr>
+                            <td>{{ forloop.counter }}</td>
```

Comment:
what will this display with empty top 3?

Reply:
Blank table

Reply:
NOTE: Please hide top 3 if nothing should be shown